webapp-django/crashstats/crashstats/views.py

```diff
@@ -38,7 +38,7 @@
 GRAPHICS_REPORT_HEADER = (
     'signature',
     'url',
-    'uuid_url',
+    'crash_id',
```

Comment:
In https://github.com/mozilla/socorro/commit/874b6d90c1221d4f3d90ec1871fbf633d104262f I failed to update this. I don't know how I didn't notice. Simply running the report via django would fail. Perhaps I didn't run it *after* I saved my changes.